### PR TITLE
Setup components and config for container cluster when zookeeper is c…

### PR DESCRIPTION
**Pull request by hmusum**
…onfigured

Feature flag probably needed only short-term the way it is done now, but might be useful when we start testing this

### CODE REVIEW

File: config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java [hunk 1]
```diff
@@ -218,7 +218,8 @@ private void addZooKeeper(ApplicationContainerCluster cluster, Element spec) {
                                                MIN_ZOOKEEPER_NODE_COUNT + " and " + MAX_ZOOKEEPER_NODE_COUNT);
         }
         cluster.addSimpleComponent("com.yahoo.vespa.curator.Curator", null, "zkfacade");
-        // TODO: Add server component
+        cluster.addSimpleComponent("com.yahoo.vespa.curator.ReconfigurableVespaZooKeeperServer", null, "zkfacade");
```
Comment (mpolden) [inline config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java:221]: Wrong package and bundle?

Comment (hmusum) [reply config-model/src/main/java/com/yahoo/vespa/model/container/xml/ContainerModelBuilder.java:221]: Yes, will fix

**Review by mpolden — APPROVED** [review]
👍

**Comment by hmusum** [comment]
Build still pending (!?), merging

**Pull request merged**